File: CS-Examples/04_RowsColumns/DeleteMultipleRowsAndColumns.cs
using System;
using System.Windows.Forms;
using Spire.Xls;

namespace DeleteMultipleRowsAndColumns
{

	public partial class Form1 : Form
	{
        public Form1()
        {
            InitializeComponent();
        }

		private void btnRun_Click(object sender, EventArgs e)
		{
            //Create a workbook
			Workbook workbook = new Workbook();

            //Load the document from disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\CommonTemplate1.xlsx");

            //Get the first worksheet
            Worksheet sheet = workbook.Worksheets[0];

            //Delete 4 rows from the fifth row
            sheet.DeleteRow(5, 4);

            //Delete 2 columns from the second column
            sheet.DeleteColumn(2, 2);

            //Save the document
            string output = "DeleteMultipleRowsAndColumns.xlsx";
			workbook.SaveToFile(output, ExcelVersion.Version2013);

            //Launch the file
			ExcelDocViewer(output);
		}
        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

	}
}

File: CS-Examples/04_RowsColumns/DeleteMultipleRowsAndColumns.vb
Imports Spire.Xls

Namespace DeleteMultipleRowsAndColumns

	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            'Creates a new instance of the Workbook class.
            Dim workbook As New Workbook()

            'Loads the Excel document from the specified path.
            workbook.LoadFromFile("..\..\..\..\..\..\Data\CommonTemplate1.xlsx")

            'Retrieves the first worksheet from the workbook.
            Dim sheet As Worksheet = workbook.Worksheets(0)

            'Deletes four rows starting from the fifth row.
            sheet.DeleteRow(5, 4)

            'Deletes two columns starting from the second column.
            sheet.DeleteColumn(2, 2)

            'Specifies the name of the resulting Excel file.
            Dim output As String = "DeleteMultipleRowsAndColumns.xlsx"
            'Saves the modified workbook to a file with the specified name and Excel version.
            workbook.SaveToFile(output, ExcelVersion.Version2013)
            ' Release the resources used by the workbook
            workbook.Dispose()
            'Launch the file
            ExcelDocViewer(output)
		End Sub
		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace
